src/main/java/uk/gov/hmcts/ccd/data/roleassignment/RoleAssignmentRecord.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.ccd.data.roleassignment;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+
+@Builder
+@Data
+public class RoleAssignmentRecord {
+    private String id;
+    private String actorIdType;
+    private String actorId;
+    private String roleType;
+    private String roleName;
+    private String classification;
+    private String grantType;
+    private String roleCategory;
+    private Boolean readOnly;
+    private String beginTime;
```

Comment:
considering we'll have to use beginTime and endTime to filter role assignments, maybe better declare these with a date type?